docs/apply/object/move_or_destruct.md

```diff
@@ -9,16 +9,27 @@ title: object / move_or_destruct
 
 ### SYNOPSIS
 
-    int move_or_destruct( object dest );
+    void move_or_destruct( object dest );
 
 ### DESCRIPTION
 
-    If  an  object's is destructed, prior to it's destruction this apply is
-    called on it's contents.  'dest' will be the to be  destructed  object.
-    If  the  target  object  of  this apply does not move itself out of the
+    If an object is destructed, prior to its destruction this apply is called
```

Comment:
This first sentence still feels like a mouthful with the weird comma placement and double destruction word. I suggest something more like:
```
When an object is destructed, this apply is called on each of its inventory contents.
```